src/lib.rs

```diff
@@ -398,6 +403,23 @@ impl purple::InputHandler for PurpleICQ {
     }
 }
 
+impl purple::CommandHandler for PurpleICQ {
+    fn command(
+        &mut self,
+        conversation: &mut ChatConversation,
```

Comment:
TODO: Switch a Conversation quand je merge mon autre branch #37

Reply:
yeah je l'avais noté